content/en/scale-agent/tasks/dynamic-accounts/manage-accounts.md

```diff
@@ -0,0 +1,237 @@
+---
+title: Manage Kubernetes Accounts in the Armory Scale Agent
+linkTitle: Manage Accounts
+description: >
+  Learn how to create, update, delete, and fetch accounts in the Armory Scale Agent for Spinnaker and Kubernetes.
+---
+
+## Manage accounts overview
+
+{{< include "scale-agent/api-overview" >}}
+
+See {{< linkWithTitle "scale-agent/tasks/dynamic-accounts/migrate-accounts.md" >}} for details on how to manually or automatically migrate accounts from Clouddriver to the Scale Agent using the Dynamic Accounts API.
+
+## Create accounts
+
+Use this when you want to create an account that does not exist in Clouddriver or the Scale Agent.
```

Comment:
Let's mention that the account will be in state INACTIVE and that Patching to ACTIVE is required to start using it, and maybe we can reference the other section we already have or include the PATCH to ACTIVE operation here.